lib/views/general/_responsive_topnav.html.erb

```diff
@@ -14,7 +14,7 @@
         </li>
 
 
-        <li><%= link_to _("Chi Siamo"), ' http://www.dirittodisapere.it/chi-siamo/' %></li>
+        <li><%= link_to "Chi Siamo", ' http://www.dirittodisapere.it/chi-siamo/' %></li>
         <li class="<%= 'selected' if params[:controller] == 'help' %>">
           <%= link_to _("Help"), help_about_path %>
         </li>
```

Comment:
Search box looks broken:

![screen shot 2016-06-23 at 09 55 43](https://cloud.githubusercontent.com/assets/282788/16297412/abd4e0f0-3928-11e6-8dff-c1b21afa27e7.png)

Doesn't seem to be a consequence of updated (longer) translations (edited in web inspector).

Also, the search form method needs to be "GET" as per https://github.com/mysociety/alaveteli/blob/master/doc/CHANGES.md#upgrade-notes-11